13/alpine/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM alpine:3.9
```

Comment:
3.10?

Reply:
Good idea.

Reply:
but I'm not sure when the config is still 3.9, which means it'll be a manual edit

Reply:
I think this should be a manual edit because we are really close to support multiple versions of alpine anyways.

Reply:
Should be done